extensions/wikia/PhalanxII/js/SpecialPhalanx.js

```diff
@@ -2,6 +2,9 @@ require(['jquery', 'mw', 'phalanx', 'BannerNotification'], function($, mw, phala
 	// edit token is required by Phalanx API
 	phalanx.init(mw.config.get('wgPhalanxToken'));
 
+	var $filter = $('#wpPhalanxFilter'),
+		$bulkFilter = $('#wpPhalanxFilterBulk');
+
 	$('body').
 		// handle blocks "unblocking" (i.e. removing blocks)
 		on('click', 'button.unblock', function(ev) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "ev". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)